.travis.yml

```diff
@@ -33,12 +39,79 @@ matrix:
         env: DJANGO_VERSION_CEILING=2.1 DJANGO_DB_ENGINE=postgres DJANGO_DB_NAME=testdb DJANGO_DB_USER=postgres
       - python: 3.6
         env: DJANGO_VERSION_CEILING=2.1 DJANGO_DB_ENGINE=mysql DJANGO_DB_NAME=testdb DJANGO_DB_USER=travis
-
-
+      - python: 3.6
+        env: DJANGO_VERSION_CEILING=2.2 DJANGO_DB_ENGINE=sqlite
+      - python: 3.6
+        env: DJANGO_VERSION_CEILING=2.2 DJANGO_DB_ENGINE=postgres DJANGO_DB_NAME=testdb DJANGO_DB_USER=postgres
+      - python: 3.6
+        env: DJANGO_VERSION_CEILING=2.2 DJANGO_DB_ENGINE=mysql DJANGO_DB_NAME=testdb DJANGO_DB_USER=travis
+      - python: 3.7
+        env: DJANGO_VERSION_CEILING=1.12 DJANGO_DB_ENGINE=sqlite
+        dist: xenial
+        sudo: true
+        services:
+          - mysql
+          - postgresql
+      - python: 3.7
+        env: DJANGO_VERSION_CEILING=1.12 DJANGO_DB_ENGINE=postgres DJANGO_DB_NAME=testdb DJANGO_DB_USER=postgres
+        dist: xenial
+        sudo: true
+        services:
+          - mysql
+          - postgresql
+      - python: 3.7
+        env: DJANGO_VERSION_CEILING=1.12 DJANGO_DB_ENGINE=mysql DJANGO_DB_NAME=testdb DJANGO_DB_USER=travis
+        dist: xenial
+        sudo: true
+        services:
+          - mysql
+          - postgresql
+      - python: 3.7
+        env: DJANGO_VERSION_CEILING=2.1 DJANGO_DB_ENGINE=sqlite
+        dist: xenial
+        sudo: true
+        services:
+          - mysql
+          - postgresql
+      - python: 3.7
+        env: DJANGO_VERSION_CEILING=2.1 DJANGO_DB_ENGINE=postgres DJANGO_DB_NAME=testdb DJANGO_DB_USER=postgres
+        dist: xenial
+        sudo: true
```

Comment:
for internal repos, I did: `sudo: required` based on talking to Travis's support.
also,m we can put the dist & sudo stuff top level instead of just putting it for 3.7.

Reply:
Won't that move every test to a fully virtualized env instead of using containers, thus slowing test runs?

Reply:
maybe... but I don't think this actually matters since we don't really update this repo that much